### ISSUE TITLE
Table 3 Training Configuration

### ISSUE THREAD

**Issue by ovshake** [jonmun/MM-SADA-code#7]
Hi, 
Thanks for the wonderful work and for making it open-source. Could you please let me know the training configuration (LR, no. of epochs, decay rate, batch size) for the table 3 results, particularly the source only RGB baseline? Is it the same as the MM-SADA experiments i.e 9k epochs? 
Also, in table 3, the result for source only RGB D1 -> D3 is given 36.6 but in Figure 5(c), the green dotted line definitely more than 36.6 (looks more near to 40). Could you shed some light on this? 
![image](https://user-images.githubusercontent.com/22028532/130217301-0fea9813-04e9-4e40-9e51-bac99bfc6071.png)

![image](https://user-images.githubusercontent.com/22028532/130216978-fadef4ed-f3e0-4d7d-ae82-dae00a424faf.png)

Thanks in advance

**Comment by jonmun** [comment]
Hi, thank you for your interest in the work.

The RGB baseline is simply an evaluation of the RGB classifier with weights learned by MM-SADA. To replicate the results change `--modality="joint"` to `--modality="RGB"` in test submission script: `scripts/submit_test.sh`.

The training configuration is the same as in the paper, and can also be found in the submission scripts: `scripts/*.sh`

Fig 5 shows the performance of the multi-modal classifier (late fusion of RGB + Flow), whereas, Table 3 is the performance of individual modalities. The RGB accuracy should be lower than the multi-modal classifier.

Best,
Jonathan

**Comment by ovshake** [comment]
Thanks for the reply, Jonathan. Did you use i3d pretrained on kinetics400 to get the source only numbers? I am asking because this [line](https://github.com/jonmun/MM-SADA-code/blob/69fe0f71ec795677461e0ed9383ea913c7c5425c/scripts/submit_test.sh#L7) implies kinetics pretrained was used (I think?) but the paper didn't have any mention of Kinetics dataset.

**Comment by jonmun** [comment]
Yes, we used the kinetics400 pretrained weights for all our models.
